src/libraries/tests.proj

```diff
@@ -6,6 +6,10 @@
     <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup Condition="'$(RuntimeFlavor)' == 'Mono'"> 
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)\System.Utf8String.Experimental\tests\System.Utf8String.Experimental.Tests.csproj" />
+  </ItemGroup>
```

Comment:
Is this the best / only place to do this?  We've built the engineering system here such that core files like this one don't need to know about individual test projects, but now we're adding a specific suppression to this general file.  There's no way to do this as part of System.Utf8String.Experimental?

Reply:
The project shouldn't be loaded at all, hence excluding it in the traversals (tests.proj, src.builds, ref.builds, packages.builds) is fine.

Reply:
I understand, but why can't something be put into the System.Utf8String.Experimental directory that would prevent that as well?

Reply:
We currently just glob on csproj files and invoke them in tests.proj and other traversals. We could do an exclusion check based on a file being present in every item's project dir but I doubt that's worth the hassle. Just doing the file globbing in msbuild is already slow.

Reply:
Then how bad is it if we just load the project?  We're talking about one project out of hundreds that will get loaded and ignored in one build.

Reply:
I'm sure @EgorBo can try that out. To be consistent, the same needs to be done for the ref and the src project which are also excluded.

Reply:
I'd appreciate if we could merge this and file an issue to make it better to unblock our test build :)

Reply:
I am not a build system expert, how can I disable build for certain libraries in their folders? what property can I use in `Directory.Build.props` ?

Reply:
> I'd appreciate if we could merge this and file an issue to make it better to unblock our test build :)

Ok.

Reply:
Filed https://github.com/dotnet/runtime/issues/34499